content/blog/2022-05-17-aws-remotes-in-dvc.md

```diff
@@ -0,0 +1,158 @@
+table. Note that you won’t be able to access your secret key again at this
+point. You would need to make a new set of credentials if you don’t have it.
+
+![successfully created AWS access credentials](/uploads/images/2022-05-17/credentials.png)
+
+That’s it for setting up your bucket and getting the credentials you need! Now
+let’s add DVC to our demo repo and set up the remote.
+
+## Set up a DVC project
+
+First, add DVC as a requirement to your project with the following installation
+command:
+
+```dvc
+$ pip install 'dvc[s3]'
+```
+
+Then you can initialize DVC in your own project with the following command:
+
+`$ dvc init`
+
+This will add all of the DVC internals needed to start versioning your data and
+tracking experiments. Now we need to set up the remote to connect our project
+data stored in AWS to the DVC repo.
+
+### Create a default remote
+
+Now we can add a default to the project with the following command:
+
+`$ dvc remote add -d bikes s3://updatedbikedata`
+
+This creates a default remote called `bikes` that connects to the
+`updatedbikedata` bucket we made earlier which is where the training data for
+the model will be stored.
+
+### Add AWS credentials
+
+In order for DVC to be able to push and pull data from the remote, you need to
+have valid AWS credentials.
+
+By default, DVC authenticates using your AWS CLI configuration, if it has been
+set. You can check that out
+[here in the AWS docs](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html).
+
+If you want to
+[use a different authentication method](https://dvc.org/doc/command-reference/remote/modify#amazon-s3)
+or if you run into issues with the credentials, you can manually add them with
+the following commands:
+
+`$ dvc remote modify --local bikes access_key_id 'mykey'`
+
+`$ dvc remote modify --local bikes secret_access_key 'mysecret'`
+
+### Push and pull data with DVC
+
+Now you can push data from your local machine to the AWS remote! First, add the
+data you want DVC to track with the following command:
+
+`$ dvc add data`
+
+This will allow DVC to track the entire `data` directory so it will note when
+any changes are made. Then you can push that data to your AWS remote with this
+command:
+
+`$ dvc push`
+
+If you're curious about how DVC works with data from your local cache in the
+remote, you can learn more in
+[the docs here](https://dvc.org/doc/command-reference/push#example-what-happens-in-the-cache).
+You can also learn more about the
+[structure of the cache directory here](https://dvc.org/doc/user-guide/project-structure/internal-files#structure-of-the-cache-directory).
+Here's what the data might look like in your AWS bucket.
+
+![data in AWS bucket](/uploads/images/2022-05-17/aws_bucket.png)
+
+Then if you move to a different machine or someone else needs to use that data,
+it can be accessed by cloning the project repo, connecting to the remote, and
+running:
```

Comment:
```suggestion
it can be accessed by cloning (or pulling) the repo and running:
```

Instead of the "connecting to the remote" part maybe a note after the sample block mentioning authentication has to be setup locally in each machine (link to **Add AWS credentials**?)